packages/safe-core-sdk/src/Safe.ts

```diff
@@ -453,6 +466,49 @@ class Safe {
     return ownersWhoApproved
   }
 
+  /**
+   * Returns the Safe transaction to enable a Safe guard.
+   *
+   * @param guardAddress - The desired guard address
+   * @param options - The transaction optional properties
+   * @returns The Safe transaction ready to be signed
+   * @throws "Invalid guard address provided"
+   * @throws "Guard provided is already enabled"
+   * @throws "Current version of the Safe does not support Safe transaction guards functionality"
+   */
+  async getEnableGuardTx(
+    guardAddress: string,
+    options?: SafeTransactionOptionalProps
+  ): Promise<SafeTransaction> {
+    const safeTransaction = await this.createTransaction({
+      to: this.getAddress(),
+      value: '0',
+      data: await this.#guardManager.encodeEnableGuardData(guardAddress),
+      ...options
+    })
+    return safeTransaction
+  }
+
+  /**
+   * Returns the Safe transaction to disable a Safe guard.
+   *
+   * @param guardAddress - The desired guard address
+   * @param options - The transaction optional properties
+   * @returns The Safe transaction ready to be signed
+   * @throws "Invalid guard address provided"
+   * @throws "There are no guards enabled already"
+   * @throws "Current version of the Safe does not support Safe transaction guards functionality"
+   */
+  async getDisableGuardTx(options?: SafeTransactionOptionalProps): Promise<SafeTransaction> {
+    const safeTransaction = await this.createTransaction({
+      to: this.getAddress(),
+      value: '0',
+      data: await this.#guardManager.encodeDisableGuardData(),
+      ...options
```

Comment:
Just to confirm, it's a conscious decision to allow the options to potentially override the three props above?

Reply:
`to`, `value` and `data` are not included `options` type